static/src/javascripts/projects/commercial/modules/dfp/render-advert.js

```diff
@@ -33,23 +32,6 @@ const addClassIfHasClass = (newClassNames: Array<string>) =>
                     newClassNames.forEach(className => {
                         advert.node.classList.add(className);
                     });
-                    // Add fluid styles from _adslot.scss
```

Comment:
Seems like there was a bug in there, good catch! 👍 (this block was there to mirror the `.ad-slot--fluid` css rules into DCR.)

 I think it comes down to the fact that `newClassNames` don't have the `ad-slot--im` class in it when called in [here](https://github.com/guardian/frontend/blob/f3a6d6a7894703798ab7d3e885aa665f955f15d7/static/src/javascripts/projects/commercial/modules/dfp/render-advert.js#L60) 

Could we then make sure [these rules](https://github.com/guardian/frontend/blob/ba6ff947c2551f98da59dc7252dda78bc95c6a4e/static/src/stylesheets/module/_adslot.scss#L413) are properly migrated into DCR, potentially with some global styles?

Reply:
I feel like some of the styles are missing from https://github.com/guardian/dotcom-rendering/pull/2017